### ISSUE TITLE
No module named 'vggt'

### ISSUE THREAD

**Issue by bqm1111** [vita-epfl/VoxDet#6]
When I tried to retrain model, I ran into this error "No module named 'vggt'". How can I install vggt?

**Comment by wymanCV** [comment]
Hi, thank you for your interest! Sorry, the previous version we sent to others includes VGGT for follow-up developing usage. I have just updated them. Please check commit #19, and you can play with it in the occupancy. Thanks!

**Comment by bqm1111** [comment]
Thank you for your quick response. May I ask what is your follow up plan using VGGT?

**Comment by bqm1111** [comment]
Also. I only have 2 GPUs with 24GB VRAM each so I have to decrease the batch size to 1. What is the most efficient way to train the model if I only have 1 GPU to use?

**Comment by wymanCV** [comment]
Hi, we previously tried some feature distillation and estimated camera parameters with VGGT, but did not follow up due to the lack of novelty.

It may influence the performance a lot due to the small batch-size..  You may need to train the model longer (~2x/4x iterations) and adjust some common hyperparameters, e.g., learning rate/schedule accordingly. Hope this can help. Thanks.

**Comment by bqm1111** [comment]
Can you explain why your method consumes so much memory during training? To my knowledge, methods like VoxFormer only consume less than 16GB during training.

**Comment by wymanCV** [comment]
We use similar memory (<20G). The potential performance decrease is mainly caused by batch-size but not the memory. But you can have a try and may obtain very comparable results when training longer!

**Comment by bqm1111** [comment]
> Hi, we previously tried some feature distillation and estimated camera parameters with VGGT, but did not follow up due to the lack of novelty.
> 
> It may influence the performance a lot due to the small batch-size.. You may need to train the model longer (~2x/4x iterations) and adjust some common hyperparameters, e.g., learning rate/schedule accordingly. Hope this can help. Thanks.

May I ask why it lacks novelty? Developing an end-to-end model that is agnostic to camera parameters sounds quite interesting to me.

**Comment by wymanCV** [comment]
I remember that some existing works have done this (not SSC, but the multi-view occupancy prediction). Anyway, they did not use foundational models, and you can give it a try!
